zendesk/data_source_zendesk_system_field.go

```diff
@@ -0,0 +1,142 @@
+package zendesk
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/nukosuke/go-zendesk/zendesk"
+	"strconv"
+)
+
+func dataSourceZendeskSystemField() *schema.Resource {
+	return &schema.Resource{
+		Read: func(data *schema.ResourceData, i interface{}) error {
+			zd := i.(*zendesk.Client)
+			return readSystemField(data, zd)
+		},
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
```

Comment:
potentially  should rename this to something like ticket field id

Reply:
If this `data` would be renamed to `ticket_field`, it would be clear what the name, `id` means.
then, I think this is fine like this.